src/auto/functions.rs

```diff
@@ -603,6 +615,12 @@ pub fn unix_is_system_fs_type(fs_type: &str) -> bool {
 //    unsafe { TODO: call gio_sys:g_unix_mount_get_options() }
 //}
 
+//#[cfg(any(unix, feature = "dox"))]
+//#[cfg(any(feature = "v2_60", feature = "dox"))]
+//pub fn unix_mount_get_root_path(mount_entry: /*Ignored*/&mut UnixMountEntry) -> Option<GString> {
```

Comment:
`UnixMountEntry` and `UnixMountPoint` can be generated but currently used without cfg(unix),
so build fail on windows

Reply:
Ok but that can be fixed by configuring that or not?

Reply:
Currently - not,
If I even with `cfg_condition = "unix"` on `UnixMountEntry`
in used without it `use UnixMountEntry;`
see https://github.com/gtk-rs/gio/compare/master...EPashkin:unixmount?expand=1